scala/spark-pubsub/cloud-pubsub-receiver/cloudpubsubreceiver.sbt

```diff
@@ -0,0 +1,16 @@
+name := "Cloud Pubsub Receiver"
+
+version := "0.0"
+
+scalaVersion := "2.10.3"
+
+libraryDependencies ++= Seq("org.apache.spark" % "spark-core_2.10" % "1.3.0", 
+      "org.apache.spark" % "spark-streaming_2.10" % "1.3.0",
+      "org.apache.hadoop" % "hadoop-common" % "2.5.2", 
+      "org.apache.hbase" % "hbase-client" % "1.0.1", 
+      "org.apache.hbase" % "hbase-server" % "1.0.1", 
+      "org.apache.hbase" % "hbase-common" % "1.0.1",
+      "org.apache.hbase" % "hbase-annotations" % "1.0.1",
+      "org.mortbay.jetty.alpn" % "alpn-boot" % "7.0.0.v20140317",
```

Comment:
Bdutil uses 7.0.0: https://github.com/GoogleCloudPlatform/bdutil/blob/f1b5b36bae4bd8b5abb09d94d3575980b307755d/extensions/bigtable/bigtable_env.sh#L40
Should I use 7.1.3 instead even though it won't be in the VMs?